### PR TITLE
Rebuild for gfortran 7 for OSX (fixed)

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #3 
Closes #5

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

**Comment by beckermr** [comment]
@conda-forge/libmed This recipe appears to build correctly at v4 without the swig patch. IDK if it is needed or not, but if it isn't, then please meege this PR!

**Comment by looooo** [comment]
Thanks @beckermr for havin a look at this feedstock. Can you explain why you closed this PR?

**Comment by beckermr** [comment]
I was trying to get the rebuild to work but could not. :/

**Comment by beckermr** [comment]
If you understand this windows build error (https://ci.appveyor.com/project/conda-forge/libmed-feedstock/builds/27113049/job/453msgp4tu0wiblu), feel free to grab my changes and try again!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,7 +23,7 @@ requirements:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
     host:
-        - hdf5 1.8*
```
Comment (looooo) [inline recipe/meta.yaml:30]: Not sure if libmed works with latest hdf5.

Comment (beckermr) [reply recipe/meta.yaml:30]: If you update to 4.0.0, it does. However, I encountered an error on windows as indicated above.

**Pull request closed without merging**